### PR TITLE
Adds navbar

### PR DESCRIPTION
**Pull request by takshch**

### CODE REVIEW

File: main.css
Comment (RohitKuwar) [inline main.css:9]: @takshch Please remove these whitespaces after each CSS block.

**Review by RohitKuwar — CHANGES_REQUESTED** [review]

File: index.html
Comment (sumitd94) [inline index.html:11]: NIT: we can use shorthands for v-bind as :sites

File: main.css
Comment (sumitd94) [inline main.css:72]: Looks like the tab indentation is of size 4, can we please make it to 2.
Please correct me if its already 2.

File: main.js
Comment (sumitd94) [inline main.js:71]: will this be hosted on the learning site?

File: main.js
Comment (sumitd94) [inline main.js:119]: NIT: we can use the shorthand for `v-bind` :site

File: main.js
Comment (sumitd94) [inline main.js:133]: shorthand?

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few NIT Comments, Good job by the way :)

File: main.css
Comment (takshch) [reply main.css:9]: @RohitKuwar Can you please tell me why it is better to remove whitespaces or new line after each CSS block.

File: main.css
Comment (takshch) [reply main.css:72]: It is tab of 4 space. I will fix it.

File: main.js
Comment (takshch) [reply main.js:71]: This is navbarItems data which is used by navbar-item component to generate item of navbar.

File: main.css
Comment (SBagaria2710) [reply main.css:8]: I agree with Taksh here. That's how I do it as well.

File: main.js
Comment (akankshadharkar) [reply main.js:71]: I think he's referring to it being linked relatively to the current site(learn)
Which is causing the link to break since it should be redirecting to `https://www.realdevsquad.com/events.html` 

Could you fix that, please?

Comment (akankshadharkar) [reply main.js:71]: I think he's referring to it being linked relatively to the current site(learn)
Which is causing the link to break since it should be redirecting to `https://www.realdevsquad.com/events.html` 

Could you fix that, please?

**Review by akankshadharkar — CHANGES_REQUESTED** [review]
Thanks, Taksh 
Mostly lgtm 

A quick question (and need of possible change), why is "Status" being hinted at being the active element? 
Shouldn't we have a new "Learn" (or a name specific to the current site) be green instead?

Screen Shot 2021-05-21 at 5 13 52 PM

File: main.js
Comment (takshch) [reply main.js]: Fixed!

**Comment by takshch** [comment]
@akankshadharkar I added 'Learn' item in navbar with site link 'https://learn.realdevsquad.com'. 
@SBagaria2710 @RohitKuwar @sumitd94 Please approve the PR if PR looks good you.

File: main.js
Comment (akankshadharkar) [inline main.js:87]: Thanks for addressing my comment
However, this may not be the required solution 

Essentially on "Learn" we want to redirect to the page it is currently on which can be achieve with a simple <a href="/"> in the html itself

**Review by akankshadharkar — CHANGES_REQUESTED** [review]
A small comment 
Can we also have the navbar always stick at the top, even during the scroll?

File: main.js
Comment (takshch) [reply main.js]: Fixed!

Comment (akankshadharkar) [reply main.js]: Oops, I meant `href="#"`

**Comment by akankshadharkar** [comment]
Also, are we taking up responsive design in the next iteration?

File: main.js
Comment (takshch) [reply main.js]: It's href="#" now.

**Comment by takshch** [comment]
@akankshadharkar we can go for responsive design though i think it is better to work on functionalities first.

**Comment by akankshadharkar** [comment]
@takshch It need not be 100% responsive from the start but it should take into account the same from the beginning
Otherwise in the future, we'll simply be focusing on re-making the entire feature design instead of implementing new features

**Comment by akankshadharkar** [comment]
@takshch any update on the responsive design?

**Comment by takshch** [comment]
@akankshadharkar This PR is on hold for a long time. So, Please merge this PR.
I will send PR for responsive design whenever i get the time.

**Comment by akankshadharkar** [comment]
CC: @ankushdharkar

**Comment by SBagaria2710** [comment]
This has conflicts because of the new `project init` pr that got merged.
Feel free to pickup another tasks/issues :)

**Pull request closed without merging**
